TF_Tooling_Web/src/Cursors/SvgCursor.ts

```diff
@@ -3,6 +3,11 @@ import { InputType, TouchFreeInputAction } from "../TouchFreeToolingTypes";
 import { MapRangeToRange } from "../Utilities";
 import { TouchlessCursor } from "./TouchlessCursor";
 
+export interface SVGCursorParams {
+    ringSizeMultiplier: number;
+    darkCursor: boolean;
```

Comment:
Make these optional, `FromParamObj` will pass undefined on and that will use default values in the constructor if they're not included.

Side note: constructor arguments are missing types can you add them too?

Reply:
This is done in the latest commit